whereabouts.go

```diff
@@ -37,14 +36,17 @@ func NewWhereabouts(wh *workloads.WorkloadHelper) *cobra.Command {
 		Short:        "Runs whereabouts workload",
 		SilenceUsage: true,
 		PreRun: func(cmd *cobra.Command, args []string) {
-			os.Setenv("JOB_ITERATIONS", fmt.Sprint(iterations))
-			os.Setenv("POD_READY_THRESHOLD", fmt.Sprintf("%v", podReadyThreshold))
-			os.Setenv("CONTAINER_IMAGE", containerImage)
-			os.Setenv("FAST", fmt.Sprint(fast))
+			additionalVars := map[string]any{
```

Comment:
Not sure what you did here. You're running the workload twice (in `PerRun` and in `Run` with and without the parameters

Reply:
Looks like `PreRun` has been removed and this is in the actual run now.